llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -3337,11 +3343,8 @@ bool SPIRVInstructionSelector::selectGlobalValue(
   if (HasInit && !Init)
     return true;
 
-  unsigned AddrSpace = GV->getAddressSpace();
-  SPIRV::StorageClass::StorageClass Storage =
-      addressSpaceToStorageClass(AddrSpace, STI);
   bool HasLnkTy = GV->getLinkage() != GlobalValue::InternalLinkage &&
-                  Storage != SPIRV::StorageClass::Function;
+                  StorageClass != SPIRV::StorageClass::Function;
```

Comment:
Can `StorageClass` be `Function` anymore? You change it to `Static` at line 3272, right?
```suggestion
                  StorageClass != SPIRV::StorageClass::Static;
```

Reply:
That's correct, removed this dead code

Reply:
Wait, static?

Reply:
You were too fast. I changed it to Private after. My mistake.